### ISSUE TITLE
icon size based on zoom level

### ISSUE THREAD

**Issue by hermann-san** [umap-project/umap#354]
Hi there,
in my map the icons are too big on a certain zoom level because there are too many icons within a region. 

Is it possible to make icons smaller based on certain zoom level?
Can this be done on a per object basis or only for the complete map?

Can the object name at a certain zoom level be removed (i.e. not be displayed)? object or only map level?

Thanks in advance for you help.

![umap](https://cloud.githubusercontent.com/assets/7679654/18224274/291a5312-71d0-11e6-8043-3e5c26aa5bfb.png)

In the advanced  properties of an object I can see "Zoom level for automatic zooms", but I don't know how to use this. No matter what I change there, the icon size doesn't change based on zoom level.

![umap advance prop](https://cloud.githubusercontent.com/assets/7679654/18224314/6aac85c4-71d1-11e6-8307-cc53c699173b.png)

**Comment by druxstr** [comment]
> Is it possible to make icons smaller based on certain zoom level?

No, but you can either clusterize points (Layer > Type of layer > with clusters) or use a different type of icons (circle, pin). 

> Can the object name at a certain zoom level be removed (i.e. not be displayed)? object or only map level?

Not to my knowledge. But You can display _objects_ based on zoom when using remote data.

>Zoom level for automatic zooms

This sets the [zoom level](http://wiki.openstreetmap.org/wiki/Zoom_levels) displayed when you zoom on a point by using the navigation buttons on pop ups. When you click '+', the zoom displayed is set by the "Zoom level for automatic zooms".

**Comment by hermann-san** [comment]
Thanks for your suggestions. 
remote data is unfortunately not an option for me. 

changing the type of icons helps a bit, but at a certain zoom level I would still need the standard icon with the symbol because I have different types of objects and symbols help in my case.
clustered points has the same problem. It does hide the symbol.
I'd much rather have only the symbol without object name and as I zoom in, the description automatically will comes up at a certain zoom level.

Changing the symbol type depending on zoom level would also help, but I understand that this is not possible in the current uMap version.

**Comment by CelianGdfrd** [comment]
Following up on this : would love for it to be a thing, it would make wider views more pleasant.
